lesson_02/script.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+//<li>
+//<a href="1.html"><img src="bookmark.png" alt="ブックマーク" />これです</a>
+//</li>
+
+const li = document.createElement("li"); //<li></li>
+
+const a = document.createElement("a"); //<a href="1.html"></a>
+a.text = "これです";
+a.href = "1.html";
+
+const img = document.createElement("img"); //<img>
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
+
+const ul = document.getElementById("target"); //ulを取得
+ul.appendChild(li); //ulにli追加
+li.appendChild(a); //liにa追加
+a.insertAdjacentElement("beforebegin", img); //aにimg追加
```

Comment:
nits
For commit messages, it is better to write the changes. (Not the file you changed.)

You can refer to it if you like!
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716